desktop/cmp/button/RestoreDefaultsButton.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {PropTypes as PT} from 'prop-types';
+import {elemFactory, HoistComponent, XH} from '@xh/hoist/core';
+import {button} from '@xh/hoist/kit/blueprint';
+import {Icon} from '@xh/hoist/icon';
+
+/**
+ * Convenience Button preconfigured for use as a trigger for resetting user
+ * customizations. Clears all grid state and preferences, then reloads the app.
+ * Accepts props documented below as well as any supported by Blueprint's Button.
```

Comment:
This is a Hoist button so we can skip the Blueprint reference here. Ideally that provides a nice abstraction over BP implementation, so if we swapped it in the future no need to go back and change these references then.

Could drop the line entirely - I think it's relatively clear from the implementation, and the idea is to typically take the defaults and not have to think about it. (Applies just as much to any of these other convenience buttons.)